src/handlebars/partials/header.handlebars

```diff
@@ -36,7 +36,7 @@
   "sameAs": [
     "https://github.com/AdoptOpenJDK",
     "https://twitter.com/adoptopenjdk",
-    "https://www.facebook.com/AdoptaJSR",
+    "https://www.facebook.com/AdoptOpenJDK",
     "https://www.youtube.com/c/AdoptOpenJDK",
     "https://plus.google.com/u/2/b/116657919064969712773/116657919064969712773"
```

Comment:
actually google+ is dead now

Reply:
I've been told that we shouldn't remove it from structured data. The account will still be there just archived